pkg/ui/workspaces/cluster-ui/src/databases/combiners.ts

```diff
@@ -147,7 +160,7 @@ const deriveDatabaseTableDetails = (
   const normalizedPrivileges = normalizePrivileges(
     [].concat(...grants.map(grant => grant.privileges)),
   );
-  const nodes = results?.stats.replicaData.nodeIDs || [];
+  const nodes: Nodes = { kind: "node", ids: results?.stats.replicaData.nodeIDs || [] };
```

Comment:
The table views appear to already be using node IDs. Please correct me if I'm reading this incorrectly.